src/llmq/instantsend.cpp

```diff
@@ -853,8 +853,10 @@ bool CInstantSendManager::ProcessPendingInstantSendLocks()
 {
     const CBlockIndex* pBlockIndexTip = WITH_LOCK(cs_main, return ::ChainActive().Tip());
     if (pBlockIndexTip && utils::GetInstantSendLLMQType(qman, pBlockIndexTip) == Params().GetConsensus().llmqTypeDIP0024InstantSend) {
-        // Don't short circuit. Try to process deterministic and not deterministic islocks
-        return ProcessPendingInstantSendLocks(true) & ProcessPendingInstantSendLocks(false);
+        bool deterministicRet = ProcessPendingInstantSendLocks(true);
+        bool nondeterministicRet = ProcessPendingInstantSendLocks(false);
```

Comment:
nit: `Ret` suffix is usually used for function params you'd want to return back when its execution is done. It would probably be better to use `Res`/`Result` or `Ok` here.